llvm/test/Feature/Repo/repo_prunning_dependents.ll

```diff
@@ -0,0 +1,53 @@
+; This testcase tests the program repository pruning pass.
+;
+; If functions 'A' and 'B' are both dependent on function 'C',
```

Comment:
The test case references 2 different situations:
a) A->C   B->C
b) A->B->C
But it only test (b).
Can you add a test case for (a)

Reply:
Good point. A new test has been added.